dags/probe_scraper.py

```diff
@@ -101,11 +101,11 @@
         python_callable=lambda: time.sleep(60 * 30))
 
     gcp_gke_conn_id = "google_cloud_airflow_gke"
-    lookml_generator = GKEPodOperator(
+    lookml_generator_prod = GKEPodOperator(
         email=["frank@mozilla.com", "dataops+alerts@mozilla.com"],
         task_id="lookml_generator",
         name="lookml-generator-1",
-        image="gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:latest",
+        image="gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:release",
```

Comment:
you're going to tag the release, 'release'?

Reply:
i thought you were going to version them, e.g. v1.0

Reply:
Hmm, if I version them, then I have to not only tag a release but update this job. Is that what you mean?

Reply:
I agree with @haroldwoo we should version and have unique tags per release so that we can compare between release and also rollback to previous release easily if needed.